RepairsListener/UseCase/AddRepairsContractsUseCase.cs

```diff
@@ -0,0 +1,63 @@
+using RepairsListener.Boundary;
+using RepairsListener.Gateway.Interfaces;
+using RepairsListener.UseCase.Interfaces;
+using Hackney.Core.Logging;
+using System;
+using System.Threading.Tasks;
+using Hackney.Shared.Asset.Domain;
+using Microsoft.Extensions.Logging;
+using System.Text.Json;
+using RepairsListener.Infrastructure;
+
+namespace RepairsListener.UseCase
+{
+    public class AddRepairsContractsUseCase : IAddRepairsContractsUseCase
+    {
+        private readonly IRepairsStoredProcedureGateway _gateway;
+        private readonly ILogger<IAddRepairsContractsUseCase> _logger;
+
+
+        public AddRepairsContractsUseCase(IRepairsStoredProcedureGateway gateway, ILogger<IAddRepairsContractsUseCase> logger) : base()
```

Comment:
Help me out with why we're calling the super constructor when we're not inhering from anything? I'm not sure it's needed?

Reply:
Not sure why that was added. I have removed it though as I agree I don't understand the point either.